src/ml/neural_net/style_transfer/mps_style_transfer_encoding_node.m

```diff
@@ -0,0 +1,49 @@
+#include <ml/neural_net/style_transfer/mps_style_transfer_encoding_node.h>
+#include <ml/neural_net/mps_layer_helper.h>
+
+@implementation TCMPSStyleTransferEncodingNode
+
+- (instancetype) initWithParameters:(NSString *)name
+                          inputNode:(MPSNNImageNode *)inputNode
+                             device:(id<MTLDevice>)dev
+                          cmd_queue:(id<MTLCommandQueue>)cmd_q
+                         descriptor:(TCMPSEncodingDescriptor *)descriptor
+                        initWeights:(NSDictionary<NSString *, NSData *> *) weights {
+  @autoreleasepool {
```

Comment:
No need to introduce an additional autorelease pool in pure ObjC code, without specific reason

Reply:
Removed!